scripts/submit-branin.sh

```diff
@@ -8,4 +8,4 @@
 module load anaconda/3 # replace: load anaconda module
 conda activate al_new  # replace: conda env name
 cd /home/mila/c/christina.humer/activelearning # replace: location of the code
-python main.py --config-name test_branin
\ No newline at end of file
+python main.py --config-name test_branin user=chumer
```

Comment:
You can use `user=$USER` to make it general and valid for any user (provided there is a config file with the user name).

Reply:
good point